dagobah/core/core.py

```diff
@@ -690,6 +733,26 @@ def _serialize(self, include_run_logs=False, strict_json=False):
             result = json.loads(json.dumps(result, cls=StrictJSONEncoder))
         return result
 
+
+    def _implements_function(obj, functions):
+        """ Checks for the existence of a method or list of methods """
+        for expected_method in functions:
+            if not (hasattr(obj, expected_method) and
+                    callable(getattr(obj, expected_method))):
+                return False
+        return True
```

Comment:
I fixed this later on, in the next PR, which includes unit test for job-in-job where I caught it. 

But yes, good candidate for not even being in the class potentially.